tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,587 @@
5z8QstttovQisHSBsUgX1nHF3nuJIxeve5j21K0RE0MpJShDxMFYmCXqtMfI5RANos2hurULjQxPaIaJtpUNVEtTBQ4g2ppaYvJpe3EIOg1WJw9DzSuZXA2d5bL+i9r2F4bHcVpESaC9r0G5wvnma/MNwzLCwuhFYNcZR1iUBcmypU7tJ52xR9SNxSCtY7bUddnUHJKeNILdZjIqE5adao5AxGRJS/BdwYJ1YvFoMfITeQMLM+qVhM1FwPZYrWzdgnkPX3+ojWL8368PxihUrAKzFXmuKxvpRiMtMwZQPvy0vN5X3D2ZPe03Ga7CK0YBM6gk2oivW7ccK5TZ5A7D0gYvGzNTbeVupxB2fErdFJaeOPwtTBRuRomipAzqBRtDN1D2nbkFzcPNZtxBm7gDFaGidqXlrYTg/HMOIZiUJjz7+jVqjN4Ku+HjVY6A7WFM+ik1+Bc4RzAqtBTGOhIDIQQtwshjgkhTggh3tFim58SQhwVQjwmhPjUZg9sPWGiiUH/4jiSMWrvW5OKPwqa1HD77bYb1QDHP/FN1z/Rw2gxAwJn4JarziAR3mNdSaVgk9A9kBKcDXa0x6zJZpyB4zkNzsBRVySQW4SJ8laepJqsnZ/1KEJhT98eSkuzoOu1c/Zs/ixAE2cwhlcs1sqnA0ZTo6hCXTNMlNEzoZwttuZ0NiGECnwIeAlwDnhACPEFKeXRum2uAn4beJ6UckEIselhP+sRgzc9/wBjfQnOLZT4+L+eQkq5doduUDGSCPnsHFWH6iREszqjKKmF9247cAZuqYRIpRBKaM3nKiolh0R1eK20LIQer3bWE7qYMwjEoJZAbhMmahdevazvMqylI6h9fbVrx9mcLwZ7+xqbUut7DYw9y+EeTdHYkd7R1hk8ufQk+/v3h3KCQCff1MPACSnlKSmlBfw98MoV2/wS8CEp5QKAlHKaTbIcJlp7mmg2ofG65+xjKG3gSSjbHZxkVnXkgBHycQl1YaLAGTS7uwkLwbF55XKkQkS25eLaHgmjOiohrijqHdUhcxtd6WxlmMh0zVqne7sEcjsx2Nu/Fy/fOKTuxOIJxjPjpPXGvJc25vcaODOrJ5Tuyu7iXP5cy885uXSSKwavaPn6dtKJGOwGnqr7+Vz1uXquBq4WQtwrhLhfCHH7Zg8sV3bQVUGqxQpnzcisZ2hdTQzC7gyMWgI5EIOwDqkD/+4IwKuUEVEqK62OokgkfTHw4iRyzwjCRBvuM6ibTeQqfs5AGP5NyGacQaLs4mWWz9knFp/g6qHV1YZatfGsWd7gioErOLl0smnOKW/lWVqc4lX/973kvva1Nv/K7aETMWjmZ1b+SzXgKuA24GeAvxRCrFqkVwjxJiHEESHEkZkmqlpPMKRuPXaqLxCDSidiUA0Thd0Z1K1nUHH9Ez3UOQOl+qUsl1FS0akkMkuBGPg3H8HFJqYHdKsDWddBCN8ZCIFIJFo6g7VKsg8OHyRjSkrVmwHbtTm9eLq5GIz53cjO9OoAyJVDV5K38rUpqPWcWjrFaA6yp6dCeX51IgbngPqg2R7gQpNtPi+ltKWUp4Fj+OLQgJTyY1LKQ1LKQ2Nj7cdG58p2R93H9WTWs9BNpJyBLwbBYuBhFoOgA1mWK5EKE9WcQdr/SgRhiJjuIzeZMzBUA2nbKFU3ENwkiWSyZdPZWs7g4NBB0qZgSfPPg+MLx3Gkw8Hhg6u2VUdGQNNwppqIweCVgB9iWsnJxZOMLvn30fquiXb/zG2hEzF4ALhKCHG5EMIA/iOwsn3uc8ALAYQQo/hho1ObObClDofU1ROsbZA3O1DdqDgD1YCqna44VWeghVcMas7ANKMVJqrOJUpWc1RhvHO7ZOiGM6gm+DVFqxVWKIkEnrWxnEFaT9NvqcxofhXcg1MPAnDT2E2rthWKgrZjDGdqddVQOzE4OneU3UX/u6tPRFAMpJQO8Bbgq8APgM9IKR8TQvy+EOLO6mZfBeaEEEeBbwD/l5Rydb/2OlgsWwy1WeGsGetaAtMqgJYK9ygKaBhHEYXS0qCaiLIZqTBRMLE0mak6mzhn0DM24wws10JTNKRtI3R/kafge+E7g9Vi0GnnftqEp7w5bNfmyNQRdmd3szPTfK0Tfec49uTUqueHkkOMpkY5vnB81WtH545y0B4GVUXbEb7VFTta+FdK+WXgyyuee3fd3yXw69U/XWGhaHPVjvV12maTQQK5E2dQDH9ZKSyLgZSREINapZNpRitMFIhBtioGcTVR79ikMwjCRELXSahKLUykJBJNE8hlp4wr3bbOQNo2uumwpCt869y3uPf8vbzm6te03F4b34n5g8ebvnbD6A18b+Z7Dc+Zrsmx+WO8sbgLbecOhBq+m9DQFoEvliwG1+kMMtX1kAud5gzCHiICP0yEBM+NRDVR4AxExYpYmMhGMxS0VLVePXYGvaNaWrpeZyClxPGc2qA6YRgktWQtTCSSyaYJ5E7WDQ+WafUySf7rN/8rlmfxqitf1XJ7fec49tRU06qhm3bcxJncGebKy8GRh6cfxvIsdsy7GJeFcwROKMXAcjyKlsvQGusfr6Qv4V+IOhpjbRbCnzwG3xkAeHYtZxDmPoMggSwqVqTCRGbRJpnREUbgDGIx6AW17mNYtzOwq82X/gjrwBkklhPICaNpArndKIoAL+9v8+LrX8lAYoCfvfZnuW7kupbba+M7keUyXm712hc377gZ8AUg4NsXvo0mNBIX5jH2h1MMOgoTbTULJf+LuN6cQVJXUMQ6Sksj4wwA16pVE0WhA1kx7YiFiRwSGR2hVxPgcZioN9SJwXr7DGpioNSLgVpzzEoiiZtfvUpdJ2IQvO+mA8/jnhe9u+V2Afq4P4fInpxCHWgcLXH9yPVk9Sz/ev5fedG+FwFw74V7OZy+Fm/pIYx9+9fc/3YQSmfw1Lyf0d8ztL47SyEE2YTWedNZFJxBkJB1bSpuBV3RUUQof21A1bVIiWLZ0QoTFVY4gzhM1BM25QyqzZeao
```

Comment:
Like in my comment above, let's keep the demo notebook just focused on ARCTIC for now.